core/store/hashtree/binprefix/mod.go

```diff
@@ -60,9 +64,35 @@ func NewMerkleTree(db kv.DB, nonce Nonce) *MerkleTree {
 	}
 }
 
+// Load tries to read the bucket and scan it for existing leafs and populate the
```

Comment:
I'm glad there is the comment. I would've never understood that by reading the code!

Reply:
See the rename above.